OpenEdXMobile/res/layout-land/empty_screen_layout.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/empty_screen_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
shows extra white space for the higher screen height devices.
<img src="https://user-images.githubusercontent.com/43750646/211803040-21d92d34-f2ea-4433-bcec-74aadac6c42a.png" height="512"/>